sdk/monitor/monitor-query/README.md

```diff
@@ -53,6 +53,13 @@ const credential = new DefaultAzureCredential();
 const logsQueryClient = new LogsQueryClient(credential);
 // or
 const metricsQueryClient = new MetricsQueryClient(credential);
+// or
+const batchEndPoint: string = "<YOUR_METRICS_ENDPOINT>"; //for example, https://eastus.metrics.monitor.azure.com/
+
+const metricsBatchQueryClient: MetricsBatchQueryClient = new MetricsBatchQueryClient(
```

Comment:
Please add the types to the LogsQueryClient and MetricsQueryClient above, so that they match your syntax for MetricsBatchQueryClient